src/driver/franka_plan_runner.cc

```diff
@@ -237,6 +230,9 @@ int FrankaPlanRunner::RunFranka() {
   status_ = RobotStatus::Running;  // init done, define robot as running
   bool status_has_changed {true};
 
+  // robot model for impedance control calculations
+  model_ = std::make_unique<franka::Model>(robot_->loadModel());
```

Comment:
this is the key change? cannot attempt to load model when ustopped? add comments

Reply:
Yes, this is the key change. Not sure if loading model was successful, but exiting the program once model was loaded and robot u-stopped caused an unrecoverable server error

Reply:
Still need to test and make sure it behaves OK if the robot is u-stopped AFTER init, and then the driver is killed